internal/pubsub/client.go

```diff
@@ -861,3 +801,24 @@ func (c *Client) canAccessTopic(ctx context.Context) error {
 
 	return nil
 }
+
+func transportCredentials(skipVerify bool) credentials.TransportCredentials {
+	if skipVerify {
+		return insecure.NewCredentials()
+	}
+
+	if pool, err := x509.SystemCertPool(); err == nil {
+		return credentials.NewClientTLSFromCert(pool, "")
+	}
+
+	return credentials.NewClientTLSFromCert(x509.NewCertPool(), "")
+}
+
+// checks connection error.
+func connErr(err error) bool {
```

Comment:
Nitpick: maybe rename to `isUnavailableErr`.

Reply:
Fixed in 6340803

Reply:
Fixed in 6340803